lib/gollum-lib/filter/toc.rb

```diff
@@ -70,7 +70,7 @@ def process(data)
   # Prefixes duplicate anchors with an index
   def generate_anchor_name(header)
     name = header.content
-    level = header.name.gsub(/[hH]/, '').to_i
+    level = header.name[1..-1].to_i
```

Comment:
Just for my understanding: what is this doing? Can you provide an example?

Reply:
`header.name` is e.g. `h2` (set by nokogiri). The gsub was removing the h character in order to get the header level. `[1..-1]` does the same but in a more efficient way:

```ruby
2.4.1 :001 > 'h12'[1..-1]
 => "12" 
2.4.1 :002 > 'H3'[1..-1]
 => "3" 
```